src/cascadia/TerminalControl/SearchBoxControl.cpp

```diff
@@ -0,0 +1,144 @@
+﻿//
+// SearchBoxControl.cpp
+// Implementation of the SearchBoxControl class
+//

```

Comment:
Change to cpp copyright header from our other files.

Reply:
added